config/manifests/bases/update-service-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |
+      [
+         {
+           "apiVersion": "updateservice.operator.openshift.io/v1",
+           "kind": "UpdateService",
+           "metadata": {
+             "name": "update-service-instance"
+           },
+           "spec": {
+             "replicas": 1,
+             "releases": "openshift-release-dev/ocp-release",
+             "graphDataImage": "quay.io/cincinnati/demo-graph-data:for-testing-only"
+           }
+         }
+       ]
+    capabilities: Basic Install
+    description: Creates and maintains an OpenShift Container Platform Update Service instance
+    operators.operatorframework.io/builder: operator-sdk-v1.2.0
```

Comment:
I think we want [`operatorframework.io/suggested-namespace: openshift-update-service`][1] in here too.

[1]: https://github.com/openshift/cincinnati-operator/pull/79/files#diff-d31bf97881e5efe5323e57c437e9999604ff7d312eef59930c415444b3e7c66aR7

Reply:
done